test/new-e2e/tests/cspm/cspm_test.go

```diff
@@ -0,0 +1,298 @@
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	localkubernetes "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/local/kubernetes"
+)
+
+type cspmTestSuite struct {
+	e2e.BaseSuite[environments.Kubernetes]
+}
+
+type findings = map[string][]map[string]string
+
+var expectedFindingsMasterEtcdNode = findings{
+	"cis-kubernetes-1.5.1-1.1.12": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.16": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.19": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.21": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.22": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.23": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.24": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.25": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.26": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.33": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.2.6": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.3.2": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.3.3": []map[string]string{
+		{
+			"result": "passed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.3.4": []map[string]string{
+		{
+			"result": "passed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.3.5": []map[string]string{
+		{
+			"result": "passed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.3.6": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.3.7": []map[string]string{
+		{
+			"result": "passed",
+		},
+	},
+	"cis-kubernetes-1.5.1-1.4.1": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-3.2.1": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+}
+var expectedFindingsWorkerNode = findings{
+	"cis-kubernetes-1.5.1-4.2.1": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-4.2.3": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-4.2.4": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-4.2.5": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-4.2.6": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-4.2.10": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+	"cis-kubernetes-1.5.1-4.2.12": []map[string]string{
+		{
+			"result": "failed",
+		},
+	},
+}
+
+func TestCSPM(t *testing.T) {
+	values := `
+datadog:
+  securityAgent:
+    compliance:
+      enabled: true`
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I prefer using go `embed` to avoid yaml formatting issues https://pkg.go.dev/embed